### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by ashraful1980** [phrutis/Rotor-Cuda#2]
Dear Sir,
It was found error; please see the bellow attached screenshot....

![image](https://user-images.githubusercontent.com/52574658/127732605-758b1a8d-f163-4f0e-b5be-95260063e89e.png)

**Comment by phrutis** [comment]
28.03.2021 у вас всё работало :)
![20210731_093740](https://user-images.githubusercontent.com/82582647/127733263-66dba044-bc6b-4cd1-b688-a1a03067b6b9.png)

**Comment by ashraful1980** [comment]
It is keyhunt cuda version 2, still it is worked...

**Comment by phrutis** [comment]
No, this is version 1.03 with random. There is no randomness in CUDA-2. The Rotor assembled on CUDA 11.4 does not work for this. In the evening I will collect on CUDA 10 should earn

**Comment by ashraful1980** [comment]
Look keyhunt cuda still worked here:

![image](https://user-images.githubusercontent.com/52574658/127734848-84b3a580-ec32-411f-ab1f-4251f50d9ab0.png)

**Comment by phrutis** [comment]
This is not a random search! The random was removed in version 1.04 and higher. A normal range search is not advisable. Read my hypothesis

**Comment by huk87** [comment]
failed at c:\users\user\downloads\keyhunt-cuda-1.03\keyhunt-cuda-1.03\keyhunt-cuda\gpu\gpuengine.cu:169
А как работать на куда 1.03 если уже более новая версия куды? Можно сверху накатить?

**Comment by phrutis** [comment]
Ты путаешь версию программы с версией cuda toolkit
